mibs/awplus/mib_info.txt

```diff
@@ -0,0 +1,5 @@
+This folder contains MIBs for Allied Telesis products running AW+ 5.x.x

```

Comment:
imho you can get rid of this file.

Reply:
OK - no problem.

I created it to try clear up why some mibs have the same names between allied folder, and awplus. It's because despite the names being the same, the content of said mibs differ slightly.

I just thought someone might come along later and go "why are there duplicate files here?".  You probably noted I put more detail in the txt file in the "allied" mibs folder also.

Reply:
I'm not sure it's needed. Anything in a sub folder is in essence after the mib dir revamp. Everything before was in one folder, now we are picky about how they are done.

Reply:
kk